packages/orchestration/src/cosmos-api.ts

```diff
@@ -22,7 +22,7 @@ import type { AmountArg, ChainAddress, DenomAmount } from './types.js';
 
 /** An address for a validator on some blockchain, e.g., cosmos, eth, etc. */
 export type CosmosValidatorAddress = ChainAddress & {
-  // TODO document why this is the format
+  // infix for Validator Operator https://docs.cosmos.network/main/learn/beginner/accounts#addresses
```

Comment:
We might consider 1) noting this is `valoper` conventionally, 2) moving this value to `ChainInfo` keyed as `bech32PrefixValPub`

https://github.com/cosmos/cosmos-sdk/blob/159584ad419e06cfd387adeb74665bfadc614df0/types/address.go#L22-L28

https://github.com/cosmos/cosmos-sdk/blob/159584ad419e06cfd387adeb74665bfadc614df0/types/address.go#L66-L67